lib/generateClass.js

```diff
@@ -276,6 +284,53 @@ function writeToJson(def, cw) {
   cw.ln(`return inst;`);
 }
 
+/**
+ * Generates a FHIR serializer for the element
+ * @param {DataElement} def - The definition of the SHR element to generate a serializer for
+ * @param {Object} fhir - The fhir export
+ * @param {CodeWriter} cw - The CodeWriter instance to use during generation
+ * @private
+ */
+function writeToFhir(def, fhir, cw) {
+
+  const fhirProfile = fhir.profiles.find(p => p.id === fhirID(def.identifier));
+
+  cw.ln(`let inst = {};`);
+  if(fhirProfile !== undefined){
+    cw.ln(`inst['resourceType'] = '${fhirProfile.type}';`);
+
+   /* UNCOMMENT BELOW CODE TO JAM IN DUMMY MAPPING DATA */ 
```

Comment:
Do we want to remove this comment, now that the data is being added by the FHIR spec exporter?

Reply:
I'll just update it in the next PR I open (as long as i'm out 'ahead' of your stuff, it helps having a reference).

Reply:
👍